### ISSUE TITLE
All dialogue missing 

### ISSUE THREAD

**Issue by 825i** [JackitK/goldensun-dawn-of-djinn#1]
I've patched this 3 times now. All dialogue is gone.

The game starts from the farewell scene after the prologue. The only dialogue box shown ever is when everyone shouts "Farewell!"

All the scripted animations and events happen but it's like someone is speeding through the dialogue so fast I can't see it. If I try and talk to any NPC nothing happens. They never say a word.

For example I go to talk to Flint for a Djinn tutorial and he's just immediately absorbed. I try and go to the first town after Vale and the caravan leaves and all these events where they say they have to take a detour to Dodonpa's place. There's no text. It's super weird.

Patched the USA, EUROPE version of the rom. Can't find any others.
I used patch v1.1.4

Another interesting thing I've found is that the patch changes the .gba rom filesize from 8MB to 16MB. Which it shouldn't do at all. Other patches I've applied don't change the filesize basically at all. Eg. if I patch Golden Sun Reloaded, the filesize doesn't change but the patch works flawlessly.

Any help would be greatly appreciated.

**Comment by JackitK** [comment]
TLDR: Dawn of Djinn is built in with a prolouge/Djinn Tutorial skip and dialogue skip. There are no plans to change this for version 1.

See the Read Me FAQ section for full details.
TLDR:
I made it dialogueless, not realizing I couldn't reverse it until late in development. The only way I know to fix this is remaking from scratch, which is why version 2 is currently being made.

Version 2 will also likely also skip the prolouge and Djinn tutuorial, as assests from that are used for Dawn of Djinn exclusive assests, such as additional new enemies. I also just prefer the game with the prolouge skip and plan to adjust the game's dialogue to adjust for the lack of it when version 2 is complete.
